mudlet-mapper.xml

```diff
@@ -8423,6 +8473,34 @@ if not alreadyRun then mmp.updateIsles() end</script>
                             <string>mmapper changed continent</string>
                         </eventHandlerList>
                     </Script>
+                    <Script isActive="yes" isFolder="no">
+                        <name>Shackle handling</name>
+                        <packageName></packageName>
+                        <script>local shackle = shackle or {density = false}
```

Comment:
why not just `local shackle = false`? (or `density`, if you prefer)

Reply:
umm I think it was because of something that I was doing before I got it working correctly that was messing with it and kept setting the table back to default and that was the fix that got it working. I'll redo it an make sure it works right when I do.